model/flow/identifier.go

```diff
@@ -121,6 +121,14 @@ func MakeID(entity interface{}) Identifier {
 	return id
 }
 
+func MakeIDFromBytes(data *[]byte) Identifier {
+	//bs2 := binstat.EnterTimeVal(binstat.BinMakeID+".??lock.Hash", int64(len(data)))
+	var id Identifier
+	hash.ComputeSHA3_256((*[32]byte)(&id), *data)
```

Comment:
The signature of `ComputeSHA3_256` is
```golang
func ComputeSHA3_256(result *[HashLenSHA3_256]byte, data []byte) {
``` 

We could `HashLenSHA3_256` also here:
```suggestion
	hash.ComputeSHA3_256((*[HashLenSHA3_256]byte)(&id), *data)
```